src/as-yaml.c

```diff
@@ -641,6 +641,7 @@ as_yaml_emit_sequence_from_str_array (yaml_emitter_t *emitter, const gchar *key,
 static void
 as_yaml_localized_list_helper (gchar *key, gchar **strv, yaml_emitter_t *emitter)
 {
+	g_autofree gchar *locale_noenc;
```

Comment:
`g_autofree gchar *locale_noenc = NULL;`, otherwise we'll crash if `strv == NULL`.